internal/pkg/plugin/manager.go

```diff
@@ -122,9 +142,14 @@ func (manager *Manager) _RegisterShutdownSignalHandlers() {
 	go func() {
 		exitSignal := <-signalChannel
 		log.Printf("Intercepted exit signal '%v'. Cleaning up...", exitSignal)
-
 		manager.Shutdown()
 
+		if (manager.ProfileFlag == "cpu") {
+			ProfileCleaner.Stop()
+			// PROFILER cleanup the profiler
+		} else if (manager.ProfileFlag == "memory") {
+				ProfileCleaner.Stop()
```

Comment:
funky indentation here

Reply:
in the actual code it does not look like this but I went back manually to check and reindent

Reply:
ping @sigalsax :)

Reply:
fixed :3